docs/maintenance.rst

```diff
@@ -148,3 +148,16 @@ This section contains useful links for maintainers and collaborators:
 
 - `Future of icalendar, looking for maintainer #360 <https://github.com/collective/icalendar/discussions/360>`__
 - `Comment on the Plone tests running with icalendar <https://github.com/collective/icalendar/pull/447#issuecomment-1277643634>`__
+
```

Comment:
Could you create a new PR without the old code? 

You can look up how to
- create a new branch
- use git cherry-pick to move the commits over.

Reply:
I am confused, can you explain better?

Reply:
Sure! When I look at the commits here:

https://github.com/collective/icalendar/pull/730/commits

It seems that you added the commits of the other PR into this PR.
That is not what is usually done on GitHub: Each PR should be able to stand alone, be merged and reviewed alone.

There is this one commit here:
![grafik](https://github.com/user-attachments/assets/6645247f-7889-4968-9fae-cbbd6397ccc2)

That is the only one that should go into this PR. Basically this PR should be based on the `main` branch of `collective/icalendar` and go to the `main` branch of `collective/icalendar` with one additional commit, nothing else.

This way, we do not get confused and re changes only show up in one PR and do not need addressing in different placed. This is also an old library, so it is good to keep a bit of discipline because we might need to know why changes happened 10 years later.

Reply:
You should be able to do this:

```
git checkout a27b146f50c1ddd83117302372d0704ce8ff0853 # the commit
git checkout -b new-branch-for-PR
git push -u origin new-branch-for-PR
```

And create a new PR without the other things.

Reply:
Okay @niccokunzmann I understand now, I'll work on that.